src/toolbar_widget.ts

```diff
@@ -12,8 +12,8 @@ export class ToolbarModel extends DOMWidgetModel {
             _view_name: 'ToolbarView',
             _model_module: 'jupyter-matplotlib',
             _view_module: 'jupyter-matplotlib',
-            _model_module_version: '^' + MODULE_VERSION,
-            _view_module_version: '^' + MODULE_VERSION,
+            _model_module_version: MODEL_VERSION,
```

Comment:
I'm not sure having the `^` does anything - and ipywidgets doesn't do this, so I removed it to be more similar to ipywidgets.